org.gnome.Music.json

```diff
@@ -20,13 +20,6 @@
         "--filesystem=xdg-music",
         "--filesystem=xdg-cache/media-art"
     ],
-    "build-options": {
-        "cflags": "-O2 -g",
-        "cxxflags": "-O2 -g",
-        "env": {
-            "V": "1"
-        }
-    },
```

Comment:
Why drop the cflags? Honest question. I guess the '-g' is not needed on flathub, but is the '-O2' default or something?

Reply:
@fosero It's now exported as default flags by the freedesktop runtime 18.08 which is used on GNOME Runtime 3.30